### PR TITLE
Ambr 864 jats

### PR DESCRIPTION
**Pull request by sgatchell**
*JIRA issue:* https://jira.plos.org/jira/browse/AMBR-864

## What this PR does:
Presents a MVP set of JATS tags on the Peer Review page.

## Developer Notes
The peer review XSL continues to be device-agnostic thus avoiding separate transforms for mobile and desktop. The intent is to handle any differences in CSS. To this end 77b9b9233637a5241aa3c9e72a015bf0009abca0 reconciles a needless difference between the mobile and desktop article transforms with regards to the class applied to a `<ul>`.

[See the ticket](https://jira.plos.org/jira/browse/AMBR-864?focusedCommentId=247607&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-247607) For a comparison of the article and peer review pages.

I've tested in a container with wombat referring to the dev rhino.  Example URL:
http://172.24.0.2:8080/DesktopPlosOne/article/peerReview?id=10.1371/journal.pone.0207232

If you want to ingest and test with a local rhino I can provide an ingestible zip for the above article.

# Code Author Tasks:

> This list should be finished before code review:
- [x] Testing or PO Accept notes that will assist the tester, Product Owner or reviewer are set in the ticket.

# Code Reviewer Tasks
- [ ] I read through the JIRA ticket's AC before doing the rest of the review.
- [ ] I ran the code (in the review environment or locally). I agree the running code fulfills the Acceptance Criteria as stated on the JIRA ticket. If not practical I have seen some evidence that the code does what it says (tests have passed).

### CODE REVIEW

File: src/main/webapp/WEB-INF/themes/desktop/sass/pages/_peer-review.scss [hunk 1]
```diff
@@ -43,3 +43,14 @@
 .peer-review-accordion-content {
   display: none;
 }
+span.underline {
+  text-decoration: underline;
+}
+
```
Comment (samV-plos) [inline src/main/webapp/WEB-INF/themes/desktop/sass/pages/_peer-review.scss]: I recommend removing the redundant `span` selectors--the class names are sufficient on their own.

File: src/main/webapp/WEB-INF/themes/mobile/resource/css/peer-review.css [hunk 2]
```diff
@@ -106,3 +106,15 @@
...
+span.underline {
+  text-decoration: underline;
+}
+
+span.strike {
+  text-decoration: line-through;
+}
+
+.monospace {
+  font-family: $font-face-monospace;
```
Comment (samV-plos) [inline src/main/webapp/WEB-INF/themes/mobile/resource/css/peer-review.css]: This is a CSS file, not SCSS, so this variable won't get transformed.  
Replace with:
`font-family: "Courier New", Courier, monospace;`

Comment (sgatchell) [reply src/main/webapp/WEB-INF/themes/mobile/resource/css/peer-review.css]: ah, thanks for the catch!

Comment (sgatchell) [reply src/main/webapp/WEB-INF/themes/mobile/resource/css/peer-review.css]: @samV-plos I just moved these styles to base.css, assuming we want them on all mobile pages?

Comment (samV-plos) [reply src/main/webapp/WEB-INF/themes/mobile/resource/css/peer-review.css]: yes, that's great, thanks!

**Comment by egh** [comment]
I really like the XSLT, @sgatchell . Looks great. It's not flowing correctly on mobile (emulation in chrome). Is this expected?
image

**Comment by sgatchell** [comment]
```
```

**Comment by sgatchell** [comment]
We've deferred that flow problem, and I see it in the FF emulator too. I've narrowed that overflow down to <a> tag value not being wrapped correctly, but only in the expandable content, like this one:
```
<p><a href="http://www.journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf">http://www.journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and <a href="http://www.journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf">http://www.journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a></p>
```

I've got a fix: 
```
.letter__body a {
  word-wrap: anywhere;
}

```

**Pull request merged**